### ISSUE TITLE
Broken taskbar hint/notification sound?

### ISSUE THREAD

**Issue by rikai** [Sytten/Facebook-Messenger-Desktop#39]
I'm using this on windows, i've enabled sounds, but i don't ever get a sound when i receive a message.

Additionally, the program doesn't flash in my taskbar. So, unless i have the (ugly, and easily missed) windows desktop notications enabled, i have no indicator that i've gotten a message.

**Comment by Sytten** [comment]
Sounds arent supported by NW.JS yet.
Its a good idea to make it flash, I will add it to the todo list

**Comment by rikai** [comment]
In that case, perhaps the "sounds" checkbox should be removed from the settings, as that's somewhat misleading.

**Comment by Sytten** [comment]
I know, but its a facebook created toobar (so I will have to find the html component related to it).
I will try to find a way to enable sound :)

**Comment by rikai** [comment]
Ah, my bad, i didn't realize that was a facebook created menu! As for a way [i did find this earlier](https://github.com/nwjs/nw.js/wiki/Using-MP3-%26-MP4-%28H.264%29-using-the--video--%26--audio--tags.) when i was looking around for what might be the issue...

**Comment by Sytten** [comment]
Thats interesting, I will take a look!

**Comment by rikai** [comment]
Yeah, a bit more looking around seems to imply that audio is as simple as using the <audio> tag ([if you dont need mp3/mp4 support](http://docs.nwjs.io/en/latest/For%20Developers/Enable%20Proprietary%20Codecs/))
